assets/robotest/download_tele.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# This file is used to download tele binaries.  It provides the following benefits:
+#
+#  1) Atomicity.  The TARGET is either complete or absent. (So long as
+#     BUILD_TMP is on the same filesystem as TARGET and that fs supports atomic
+#     renames). This means no half finished files for Make to trip on.
+#  2) The file can be overridden/replaced in the Makefile to allow enterprise
+#     specific functionaltiy.
+#
+# The following environment variables must be specified by the caller:
+#
+# BUILD_TMP - Used as staging while downloading images. Must be on the same filesystem as TARGET for atomic move operations.
+# TARGET - Where the tele binary should end up
+# VERSION - The tele version to download
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+TMP=$(mktemp -d -p "$BUILD_TMP")
+trap "rm -rf \"$TMP\"" exit
+
+wget --no-verbose "https://get.gravitational.com/gravity-$VERSION-linux-x86_64-bin.tar.gz" --directory-prefix "$TMP"
```

Comment:
isn't this supposed to use s3 for oss content?

Reply:
Is it?  I'm using the bits we provide via the downloads page:

https://gravitational.com/gravity/download

If we want our tele downloads to come from S3, we should change it there first -- my interest is in testing what end users will use.